examples/host_genome.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "host_genome",
-  "output_dir_s3": "s3://idseq-database/host_filter/pig/2019-02-06",
+  "output_dir_s3": "s3://idseq-database/host_filter/mosquitos/2019-10-02",
```

Comment:
I'm assuming you didn't mean to check in this file

Reply:
I meant to check in the file though not these changes to it. Thanks!